test/modules/GenericMqttTester/src/main.rs

```diff
@@ -43,7 +43,7 @@ async fn main() -> Result<()> {
 }
 
 fn init_logging() {
-    let subscriber = Subscriber::builder().with_max_level(Level::TRACE).finish();
+    let subscriber = Subscriber::builder().with_max_level(Level::INFO).finish();
```

Comment:
@vadim-kovalyov The reason I bumped this to info was to stop hyper from spitting out tons of logs. I tried to do this the another way first with the ```with_env_filter``` method passing in the string ```trace,hyper=info``` but this didn't work.

From my investigation in a console app I think the problem is because hyper isn't a direct dependency, but exists in the crate trc-client which the generic-mqtt-tester takes a dependency on. Am I right? What is the correct way I can filter this more granuarly?

Reply:
I'm not sure what is the correct way for managing different log level for different components. I have an open PBI for that for the broker :) But I'm totally fine for bumping max level to INFO.